Sources/MacroToolkit/Type.swift

```diff
@@ -174,6 +169,199 @@ public enum `Type`: TypeProtocol, SyntaxExpressibleByStringInterpolation {
+            if let attributedTypeSyntax {
+                base = base.addingAttributes(from: attributedTypeSyntax)
+            }
+            return NormalizedType(stringLiteral: base)
+            
+        case .function(let type):
+            var functionTypeSyntax: FunctionTypeSyntax
+            var attributedTypeSyntax: AttributedTypeSyntax? = nil
+            if let attributedSyntax = type._attributedSyntax {
+                functionTypeSyntax = attributedSyntax.baseType.cast(FunctionTypeSyntax.self)
+                attributedTypeSyntax = attributedSyntax
+            } else {
+                functionTypeSyntax = type._baseSyntax
+            }
+            let normalizedReturnClause = Type(functionTypeSyntax.returnClause.type).normalized()
+            let arrayOfTupleElements = functionTypeSyntax.parameters.map { tupleElement in
+                let normalizedType = Type(tupleElement.type).normalized()
+                let updatedElementType = TypeSyntax(normalizedType._syntax)
+                var newTupleElement = tupleElement
+                newTupleElement.type = updatedElementType
+                return newTupleElement
+            }
+            functionTypeSyntax.parameters = .init(arrayOfTupleElements)
+            
+            functionTypeSyntax.returnClause.type = TypeSyntax(normalizedReturnClause._syntax)
+            attributedTypeSyntax?.baseType = TypeSyntax(functionTypeSyntax)
+                
+            return .function(.init(functionTypeSyntax, attributedSyntax: attributedTypeSyntax))
+            
+        case .implicitlyUnwrappedOptional(let type):
+            var implicitlyUnwrappedOptionalTypeSyntax: ImplicitlyUnwrappedOptionalTypeSyntax = type._baseSyntax
+            var attributedTypeSyntax: AttributedTypeSyntax? = type._attributedSyntax
+            
+            let normalizedConstraint = Type(implicitlyUnwrappedOptionalTypeSyntax.wrappedType).normalized()
+            implicitlyUnwrappedOptionalTypeSyntax.wrappedType = TypeSyntax(normalizedConstraint._syntax)
+            attributedTypeSyntax?.baseType = TypeSyntax(implicitlyUnwrappedOptionalTypeSyntax)
+            
+            return .implicitlyUnwrappedOptional(.init(implicitlyUnwrappedOptionalTypeSyntax, attributedSyntax: attributedTypeSyntax))
+            
+        case .member(let type):
+            var memberTypeSyntax: MemberTypeSyntax = type._baseSyntax
+            var attributedTypeSyntax: AttributedTypeSyntax? = type._attributedSyntax
+            let normalizedBaseType = Type(type._baseSyntax.baseType).normalized()
+            
+            memberTypeSyntax.genericArgumentClause = memberTypeSyntax.genericArgumentClause?.normalized()
+            
+            memberTypeSyntax.baseType = TypeSyntax(normalizedBaseType._syntax)
+            attributedTypeSyntax?.baseType = TypeSyntax(memberTypeSyntax)
+            
+            return .member(.init(memberTypeSyntax, attributedSyntax: attributedTypeSyntax))
+            
```

Comment:
Remove blank lines after the end of case blocks.

Reply:
Fixed in 71aff30.